testing/test_basic.py

```diff
@@ -103,3 +103,26 @@ def test_MAIN():
     rr = testdir.run(sys.executable, "-m", "pytest", "-v")
     outcomes = rr.parseoutcomes()
     assert outcomes.get("passed") == 1
+
+
+def test_blocon_in_hook(testdir):
+    testdir.makeconftest("""
+import pytest
```

Comment:
You can indent this as you will, `makeconftest` ensures the indent is correct:

```python
def test_blocon_in_hook(testdir):
    testdir.makeconftest("""
        import pytest
        from twisted.internet import reactor, defer
        def pytest_configure(config):
            d = defer.Deferred()
            reactor.callLater(0.01, d.callback, 1)
            pytest.blockon(d)
    """)
    testdir.makepyfile("""
        from twisted.internet import reactor, defer
```

Reply:
Oh, thanks, was just using the original file style. I've updated the indentation of other tests too.

Reply:
Cool thanks!